CHANGELOG.md

```diff
@@ -6,11 +6,16 @@
 * Adds missing `license_number` option
 * Adds missing `receiver_liquor_license` option
 * Adds missing `VerificationDetails` object (closes #140, #141, #162)
-* Bumps `RestSharp` from `106.4.2` to `106.12.0`
 * Changes the Client delegate constructor from `internal` to `public`
 * Updates various information in the README related to thread-safety, examples, releasing, etc
 * Adds all missing dates and versions to the CHANGELOG
 
+## 2.5.1.4 (2021-11-03)
```

Comment:
We want to move to semantic versioning for this library as it's the only one that doesn't use it for whatever reason. A more appropriate version number with that in mind would be `2.6.0`.

Additionally, everything contained in the `NEXT RELEASE` section above in the CHANGELOG is already landed on the master branch and will get included in whatever you are about to release. As such, they should all be contained under the same version here in the CHANGELOG.

Reply:
Updated to use 2.6.0 and fixed up the CHANGELOG.md.